apps/game_backend/lib/game_backend/users/kaline_tree_level.ex

```diff
@@ -0,0 +1,22 @@
+defmodule GameBackend.Users.KalineTreeLevel do
+  @moduledoc """
+  Kaline Tree Levels.
```

Comment:
Let's write a little bit more here, specially since it's not intuitive what this is. We can copy paste some stuff from the game design notion perhaps.